client/components/global-notices/index.jsx

```diff
@@ -12,11 +12,14 @@ import Notice from 'components/notice';
 import NoticeAction from 'components/notice/notice-action';
 import notices from 'notices';
 import observe from 'lib/mixins/data-observe';
-import DeleteSiteNotices from './delete-site-notices';
+import DeleteSiteNotices from 'notices/delete-site-notices';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { removeNotice } from 'state/notices/actions'
 
 const debug = debugModule( 'calypso:notices' );
 
-export default React.createClass( {
+const NoticesList = React.createClass( {
```

Comment:
Should we change the declared name of the component and its class to `<GlobalNotices />` and `.global-notices` respectively, given the new file location?